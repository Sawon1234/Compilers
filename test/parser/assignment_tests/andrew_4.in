interface GoodStudent
{
	fun doHomework() : GoodGrade;
	fun askQuestion() : Knowledge;
}
fun stuff() : Integer = 3;
return "hello world";
