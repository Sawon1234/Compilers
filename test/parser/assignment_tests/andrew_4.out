interface GoodStudent < > extends Thing { fun doHomework < > ( ) : GoodGrade < > ; fun askQuestion < > ( ) : Knowledge < > ; } fun stuff < > ( ) : Integer < > return 3 ; return "hello world" ;